Annotation/declarative_golden_set.json

```diff
@@ -793,7 +854,7 @@
                     }
                 ],
                 "type": "declarative",
-                "clause": "",
+                "clause": "the lucky Portuguese Water Dog will move into the White House in April after she and President Obama take their daughters on Spring Break",

```

Comment:
need to change this to include "she hopes" and then create a nested EC for "hopes"